example/oot-checker/setup.py

```diff
@@ -30,6 +30,8 @@
     install_requires=[],
     tests_require=[],
     entry_points={
-        "cve_bin_tool.checker": ["checker_name = oot_checker.checker_name:get_version"]
+        "cve_bin_tool.checker": [
+            "checker_name = cve_bin_tool.checkers.__init__.py:get_version"
```

Comment:
```suggest
            "checker_name = oot_checker.checker_name:CurlChecker"
```
Here, it should be,
 name of checker =  path to that checker from root package:Checker Class

Reply:
Thanks for your input. Nice catch!